Entities/Industry/CTFShops/Quarry/Quarry.as

```diff
@@ -195,6 +204,8 @@ void spawnOre(CBlob@ this)
 	_ore.server_SetQuantity(!rare ? amountToSpawn : rare_output);
 
 	this.set_s16(fuel_prop, blobCount - actual_input); //burn wood
+	// reduce output if it's higher than minimal output
+	getRules().set_s32("current_quarry_output_" + this.getTeamNum(), Maths::Max(getRules().get_s32("current_quarry_output_" + this.getTeamNum()) - output_decrease, min_output));
```

Comment:
Would it look more elegant if you used `CRules::add_s32()`?

Reply:
Could be worth adding a wrapper for get current_output and another for set current output, would look much better